src/screens/CreatePost/index.js

```diff
@@ -449,11 +450,13 @@ const CreatePost = () => {
   };
 
   const postV2 = async () => {
+    setLoadingPost(true);
     if (!isEmptyMessageAllowed()) {
       showMessage({
         message: StringConstant.createPostFailedNoMessage,
         type: 'danger'
       });
+      setLoadingPost(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic here sets `loadingPost` to true before checking if the message is empty. If the message is empty, it shows an error message and sets `loadingPost` back to false. This is a good practice as it prevents the UI from being stuck in a loading state when there's an error.

```javascript
setLoadingPost(true);
if (!isEmptyMessageAllowed()) {
  showMessage({
    message: StringConstant.createPostFailedNoMessage,
    type: 'danger'
  });
  setLoadingPost(false);
  return true;
}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->